starters/cms/src/App.css

```diff
@@ -24,8 +24,17 @@ form {
     top: 0;
 }
 
-.field-input {
+select {
     width: 100%;
+}
+
+select:disabled,
+input[type="text"]:disabled,
+.ignored {
+    opacity: 0.5;
+}
```

Comment:
I wonder if we could add this to our plugin package CSS defaults

Reply:
Will make an issue for that, there might be a reason for not having them.